calico-enterprise/reference/installation/tigerastatus.mdx

```diff
@@ -0,0 +1,42 @@
+---
+description: Descriptions of Tigera Status fields
+---
+
+# Tigera status
```

Comment:
I think this should be `TigeraStatus` - it's an API name so spacing and capitalization matters. e.g., this is what shows up when users view them:

```yaml
kind: TigeraStatus
apiVersion: operator.tigera.io/v1
metadata:
  name: log-storage
status:
  . . .
```

Reply:
Fixed.

Reply:
Fixed.